globus_automate_client/cli/flows.py

```diff
@@ -918,6 +919,77 @@ def flow_action_log(
             format_and_echo(resp, verbose=verbose)
 
 
+@app.command("action-enumerate")
+def flow_action_enumerate(
+    roles: List[ActionRole] = typer.Option(
+        [],
+        "--role",
+        help="Display Actions where you have the selected role. [repeatable]",
```

Comment:
I assume the `ActionRole` enumeration provides the help on the allowed values?

Reply:
That's 100% correct:

```
  --role [created_by|monitor_by|manage_by]
                                  Display Actions where you have the selected
                                  role. [repeatable]  [default: ]
```